init.lua

```diff
@@ -1,30 +1,7 @@
 --stylua: ignore start
 local SetContent = ModTextFileSetContent
 
-local time = {GameGetDateAndTimeUTC()}
-SetRandomSeed(time[5] * time[6], time[3] * time[4])
-
-local function GenerateRandomNumber(iterations)
-	local number = ""
-	for i = 1, iterations do
-		number = number .. Random(0, 9)
-	end
-	return number
-end
-
-if ModSettingGet("user_seed") == nil then
-	ModSettingSet("user_seed", GenerateRandomNumber(30))
-	print("GENERATED USER SEED IS [" .. ModSettingGet("user_seed") .. "]")
-end
-
---[[ user seed use example:
-local _seed = ModSettingGet("user_seed"):sub(1, 10) or 0
-print(_seed)
-SetRandomSeed(_seed, _seed)
-for i = 1, 10 do
-	print(GenerateRandomNumber(8))
-end
---]]
+dofile_once("mods/noita.fairmod/files/content/user_seed/init.lua")
```

Comment:
We should probably delay this until `OnWorldInitialized` so there's actually a world seed incorporated into the `SetRandomSeed`/`Random` calls

Reply:
oh yeah OnWorldInitialized exists, i can probs shove the gameflag there
also i generate the user_seed ahead of time there so that the other contents can immediately access it